pages/docs/reference/using-gradle.md

```diff
@@ -154,49 +175,97 @@ With Gradle Kotlin DSL, configure source sets with `java.sourceSets { ... }` ins
 
 When targeting JavaScript, a different plugin should be applied:
 
-<div class="sample" markdown="1" mode="groovy" theme="idea">
+<div class="multi-language-sample" data-lang="groovy">
+<div class="sample" markdown="1" mode="groovy" theme="idea" data-highlight-only>
 
 ``` groovy
-apply plugin: "kotlin2js"
+plugins {
+    id 'kotlin2js' version '{{ site.data.releases.latest.version }}'
```

Comment:
The Kotlin/JS plugin is somewhat special because it's not yet properly published to be applied with the `plugins { ... }` DSL, so this like won't work without the workaround code in `settings.gradle` that is added by our New Project wizard:
```
pluginManagement {
    resolutionStrategy {
        eachPlugin {
            if (requested.id.id == "kotlin2js") {
                useModule("org.jetbrains.kotlin:kotlin-gradle-plugin:${requested.version}")
            }
        }
    }
}
```
I think it should be mentioned here as a part of the instruction.

Reply:
Here's a relevant issue: https://youtrack.jetbrains.com/issue/KT-20156

Reply:
+